userguide/hugo.yaml

```diff
@@ -119,6 +125,10 @@ params:
     enable: true
   drawio:
     enable: true
+  katex:
+    enable: false
+    mhchem:
+      enable: false
```

Comment:
These are the default values, right? If so, let's remove this from the config:

```suggestion
```

Reply:
> These are the default values, right?

Yes, that's correct.

> If so, let's remove this from the config:

These lines are not part of the config file any more.